### ISSUE TITLE
How to Query Data from Hetio?

### ISSUE THREAD

**Issue by Tabatabaei1999** [hetio/hetnetpy#52]
Hello, thank you for your excellent work on this project.

I am interested in querying data directly from Hetio but could not find documentation detailing how to do this programmatically. Specifically, I would like to obtain a subgraph from Hetio by constructing a query that retrieves, for example, certain drugs (e.g., Drug A, B, and C) and all associated genes. I am aware that this functionality is available on the website; however, I would like to know if similar capabilities are accessible via code.

**Comment by dhimmel** [comment]
I think you could pretty easily do this via a Cypher query at <https://neo4j.het.io/browser/>. But a few more questions to be sure

1. how many drugs are you looking to start with?
2. once you create your subgraph, what do you want to do with it? Visualize, explore interactively, export data, run network analyses, etc?

**Comment by Tabatabaei1999** [comment]
Thank you very much, Daniel, for sharing the link. I am now working with it and assuming it is suitable. I am currently focusing more on this, and I will reach out to you if I have any questions. 
Thank you very much for your responsiveness and support.

**Comment by Tabatabaei1999** [comment]
I attempted to retrieve all diseases in the dataset using the command `MATCH (n:Disease) RETURN n`, but it only returned 137 diseases. Could you confirm if this command is correct? I believe the total number of diseases globally is significantly higher than this
@dhimmel

**Comment by dhimmel** [comment]
> it only returned 137 diseases

That is correct as per the [manuscript](https://git.dhimmel.com/rephetio-manuscript/#nodes)

> We selected 137 terms from the [Disease Ontology](http://disease-ontology.org/) [[73](https://git.dhimmel.com/rephetio-manuscript/#ref-1GnETgp1H),[74](https://git.dhimmel.com/rephetio-manuscript/#ref-f99q1xsw)] (which we refer to as DO Slim [[75](https://git.dhimmel.com/rephetio-manuscript/#ref-203l6hC3),[76](https://git.dhimmel.com/rephetio-manuscript/#ref-rV9FuvXN)]) as our **disease** set. Our goal was to identify complex diseases that are distinct and specific enough to be clinically relevant yet general enough to be well annotated. To this end, we included diseases that have been studied by GWAS and cancer types from `TopNodes_DOcancerslim` [[77](https://git.dhimmel.com/rephetio-manuscript/#ref-bkUx1GRz)]. We ensured that no DO Slim disease was a subtype of another DO Slim disease.
